pkg/node/manager/manager.go

```diff
@@ -96,6 +98,8 @@ type AsyncOperationJob struct {
 	nodeName string
 }
 
+const pausingHealthCheckDuration = 30 * time.Minute
```

Comment:
Would it better to try a exponential back off strategy here rather than wait for a fixed amount of time? 

Most customers would recover from the failure pretty quickly I assume, we might not want to block health check for 30 mins.

Reply:
I would try to avoid more complicate exponential back off for this simple and safe skipping health check. How about 10 minutes?